src/backend/PhotoAPI/DataAccess/Implementations/ElasticService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Nest;
+
+namespace DataAccess.Implementations
+{
+    public class ElasticService : Interfaces.IElasticService
+    {
+        // FIELDS
+        private readonly string _indexName;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_indexName' or refactor the code to use its value.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=4794247)